### ISSUE TITLE
Issue with spacing in Qt and Small sizes

### ISSUE THREAD

**Issue by tobiasfalk** [hikikomori82/osifont#15]
I have posted this problem in the [Qt forum](https://forum.qt.io/topic/141142/qpainter-adds-and-removes-spaces/5) but there they say it is moslikly a problem with the font and not with Qt so here I am. Whenn I create text with Qt that is smaler than 2.4mm I get weard spacing betwen some of the chareaters.
![spcae_problems](https://user-images.githubusercontent.com/56540480/204917132-b82cd3f0-a69d-4601-b060-c075b8c54826.JPG)
![spcae_problems_2](https://user-images.githubusercontent.com/56540480/204917158-1db8db38-894a-4552-ab35-177ca6e6ef58.JPG)

**Comment by hikikomori82** [comment]
If I open your text on online demo [here](https://hikikomori82.github.io/osifont/#ABCDEF,Valid from to) and in console change font size from 8px to 1.8px and then zoom in to see it it works OK.

Make sure you are using latest version as there was kerning issues few weeks ago. Use the latest version.

**Comment by tobiasfalk** [comment]
Yes, with 1.8px it works but if you have a 480 000 DPI Page and want the text to be 1.8mm high the text has a size of 0.000 381 838 Pt. I also tried it with down to 100 DPI and threr than 1.832 82 Pt.
I also did use the newest version, partly because I was the person that mationed the last problems.
